src/internal/NeoEsp8266DmaMethod.h

```diff
@@ -61,8 +61,8 @@ struct slc_queue_item
     uint32  sub_sof : 1;
     uint32  eof : 1;
     uint32  owner : 1;
-    uint32  buf_ptr;
-    uint32  next_link_ptr;
+    uint8*  buf_ptr;
+    uint32*  next_link_ptr;
```

Comment:
I believe the better approach is to make this
```
slc_queue_item* next_link_ptr;
```
which means a review of your changes below